docs/100_producten/100_borgstelling/010_aanvragen-borgstelling/index.md

```diff
@@ -52,17 +52,13 @@ Nadat de aanvraag is goedgekeurd wordt er een borgstelling contract aangemaakt e
 Er bestaan twee soorten contracten:
 
 1. aanvraag: bruto kredietsom is verzekerd bedrag.
-2. portefuille overname: uitstaand saldo is het verzekerd bedrag en de looptijd is de resterende looptijd op basis van de uitbetaaldatum en looptijd.
+2. portefeuille overname: uitstaand saldo is het verzekerd bedrag en de looptijd is de resterende looptijd op basis van de uitbetaaldatum en looptijd.
 
 [Document](contract.message.md)
 
 ### Registreer verkoop
```

Comment:
@nielserkamp-ritense is deze nog nodig of kan de logica worden verplaatst naar het end event borgstelling afgegeven dat daar de gebeurtenis aan een wachtrij wordt toegevoegd?

Reply:
Ik zou deze vanwege duidelijkheid van het process er in laten staan aangezien het een daadwerkelijk stap in het process is en een andere taak is dan het end event.